### PR TITLE
feature flag: remove feature flag and enhance github issue link title

### PR DESCRIPTION
**Pull request by sahsisunny**
### Description:
In this PR, we have done the following changes:
- Remove feature flag to show github issue link title
- Enhance github issue link title like below:
  - Before: Issue #1234
  - After: Website Status #1234

### Anthing you would like to inform the reviewer about:
- Please review the test coverage and let us know if we need to add more test cases.

### Dev Tested:
- [x] Yes

#### Before
image

#### After
image

### Test Coverage
image
image

### Follow-up Issues (if any)
None

### CODE REVIEW

**Review by shubhamsigdar1 — DISMISSED** [review]
LGTM

File: src/utils/extractRepoName.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+function extractRepoName(issueUrl: string): string | null {
+    const regex = /https:\/\/github\.com\/([^/]+)\/([^/]+)\/issues\/(\d+)/;
+    const match = issueUrl.match(regex);
+
+    if (match && match.length === 4) {
+        const repoName = match[2]
+            .replace(/-/g, ' ')
+            .split(' ')
+            .map((word) => word.charAt(0) + word.slice(1))
+            .join(' ');
```
Comment (ajoykumardas12) [inline src/utils/extractRepoName.ts:10]: Can you please explain what line 8-10 does?

Comment (sahsisunny) [reply src/utils/extractRepoName.ts:10]:
```js
            .split(' ')    // Split on spaces
            .map((word) => word.charAt(0) + word.slice(1))  // Capitalize first letter of each word
            .join(' '); // Join words back together
```

Comment (ajoykumardas12) [reply src/utils/extractRepoName.ts:10]: Don't see capitalization part, is `toUpperCase()` missing?

**Review by ajoykumardas12 — APPROVED** [review]

**Review by shubhamsigdar1 — APPROVED** [review]

**Review by vinit717 — APPROVED** [review]

**Pull request merged**